examples/demo.js

```diff
@@ -19,34 +19,34 @@ window.onload = function() {
     for (var m in modulesInfo) {
       if (modulesInfo[m] && modulesInfo[m].name)
         addStepSelect.append(
-          '<option value="' + m + '">' + modulesInfo[m].name + "</option>"
+          "<option value=\"" + m + "\">" + modulesInfo[m].name + "</option>"
```

Comment:
Can we make it use single quotes `'`? We use a lot of HTML inside JS as strings. HTML attributes have to be wrapped in double quotes `"`. We will have to escape a lot of double quotes in our code `\"` if we use double quotes for strings and everything will look really confusing and messy. Can we instead use single quotes so that we don't have to escape double quotes?